nbs/12_optimizer.ipynb

```diff
@@ -243,7 +243,9 @@
    "source": [
```

Comment:
<p>I think this addition could be a bit distracting/confusing. Especially with the "important" at the start. For most people I don't think it's important at all! Generally speaking I don't think you'd want wd applied to normalization bias terms, would you?</p><p> </p><p>If there are situations where someone wants to make a fastai optimizer work just like pytorch's, how about we instead create a little section in this notebook called "differences to pytorch optimizers", or something like that. That way, we can remove these notes scattered through the notebook, and have all that info in one place.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/3733/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>